graphs/energy/nodes/households/households_solar_pvt_solar_radiation.ad

```diff
@@ -1,36 +1,35 @@
-# Dummy data is used for this technology, will be updated.
 - use = energetic
-- energy_balance_group = local power generation
-- output.electricity = 0.17
-- output.solar_thermal = 0.10
-- output.loss = elastic
 - groups = [cost_electricity_production, electricity_production, primary_energy_demand, sustainable_production, volatile_production, lv_net_supply, wacc_households]
 - presentation_group = solar_panel
+- graph_methods = [full_load_hours]
+- energy_balance_group = local power generation
 - merit_order.group = dynamic: solar_pv
 - merit_order.level = lv
 - merit_order.subtype = volatile
 - merit_order.type = producer
-- graph_methods = [full_load_hours]
-- availability = 0.98
-- free_co2_factor = 0.0
-- forecasting_error = 0.05
-- land_use_per_unit = 1.6000000000000001e-06
-- takes_part_in_ets = 0.0
-- part_load_efficiency_penalty = 0.1
-- part_load_operating_point = 0.7
-- electricity_output_capacity = 0.000272
-- heat_output_capacity = 0.0001
-- initial_investment = 217.328
+- output.electricity = 12.1
```

Comment:
@lottevanvlimmeren dit moeten waarden tussen de 0-1 zijn, dus delen door 100. Dit moeten we ook even aanpassen in de NSA (ik heb die alleen al gemergd ;) )

Reply:
Dit heb ik aangepast :)